kulkunen/drivers/securitas.py

```diff
@@ -0,0 +1,173 @@
+import jsonschema
+import requests
+from datetime import timedelta
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+from jsonschema.exceptions import ValidationError as JsonSchemaValidationError
+
+from .base import AccessControlDriver
+
+
+class SecuritasDriver(AccessControlDriver):
+    BASE_URL = "https://extapi.flow.securitas.com/v1/"
+
+    SYSTEM_CONFIG_SCHEMA = {
+        "type": "object",
+        "properties": {
+            "api_key": {
+                "type": "string",
+            },
+        },
+    }
+
+    RESOURCE_CONFIG_SCHEMA = {
+        "type": "object",
+        "properties": {
+            "uses_pincode": {
```

Comment:
This setting, "uses_pincode" can be set per-resource, with default being true. Some resources will not be accessible by pincode, just mobile app.